src/AppKit/Enums.cs

```diff
@@ -2154,11 +2154,201 @@ public enum NSPopUpArrowPosition : ulong {
 		Bottom
 	}
 
-	public static class NSFileTypeForHFSTypeCode {
+#if XAMCORE_4_0
```

Comment:
The bots validating all PR produce and *API diff* that is useful to see if the new code produce breaking changes
https://jenkins.mono-project.com/job/xamarin-macios-pr-builder/11308/API_20diff_20_28PR_20only_29/

In this case
```
Removed Type AppKit.NSFileTypeForHFSTypeCode
```

That's because the condition is incorrect, it should be
```csharp
#if !XAMCORE_4_0
```
(note the missing `!`) which is not defined (we're shipping `XAMCORE_3_0` so `4_0` means the next time we break API/ABI compatibility)

Reply:
Ah, I'd wondered about that one but didn't know which XAMCORE version you were shipping so I just went with the PR suggestion. Pushed the fix.